backend/prisma/migrations/20240625230356_added_peer_model/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Peer" (
+    "id" SERIAL NOT NULL,
+    "userId" INTEGER NOT NULL,
```

Comment:
Since peers are a 2-way relationship, do you plan to add two entries per relationship, with each user as `userId`?

Reply:
Yes, I do